### PR TITLE
Set up tests

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
 install:
   - npm install
+  - npm run vscode:prepublish
```
Comment (EricJizbaMSFT) [inline .travis.yml]: This does the same as `tsc -p ./` on line 24. (That being said, I would prefer you overwrite that with this `npm run` version)

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
 
 script:
   - tsc -p ./
+  - npm test --silent
```
Comment (EricJizbaMSFT) [inline .travis.yml]: Why silent?

Comment (StephenWeatherford) [reply .travis.yml]: That's what they had in https://code.visualstudio.com/docs/extensions/testing-extensions.  Will change.

File: .travis.yml (hunk 1 above)
Comment (StephenWeatherford) [reply .travis.yml]: @EricJizbaMSFT You mean this?

install:
  - npm install

script:
  - npm run vscode:prepublish
  - npm test

**Pull request merged**